filter/command.go

```diff
@@ -34,18 +33,16 @@ func (o Options) Run() error {
 	v := viewport.New(o.Width, o.Height)
 
 	var choices []string
-	if input, _ := stdin.Read(); input != "" {
-		input = strings.TrimSuffix(input, "\n")
-		if input != "" {
-			choices = strings.Split(input, "\n")
+	if len(o.Options) == 0 {
+		if input, _ := stdin.Read(); input != "" {
```

Comment:
Also here I would lean towards using a positive conditional statement so it's more readable. "If the input is empty, we do this, otherwise we use its value"

Reply:
Here o.Options is already an input, and it follows the same logic as `choose/command.go`. So I think either it should be changed in both places or kept as is, so there is a greater similarity in the structure of the two modules.